apps/razers2/outputFormat.h

```diff
@@ -785,7 +785,7 @@ void dumpMatches(
 
 		case 2:	// Eland Format
 			_sep_ = '\t';
-			for(unsigned readNo = 0; readNo < length(store.readSeqStore); ++readNo)
+			for (unsigned readNo = 0; it != itEnd; ++readNo, it++)
```

Comment:
RazerS 2 app test now fails, probably because of this change. Anyway, this app is deprecated in favor of RazerS 3, so I would't spend any time on it :walking: We can disable VS2015 in its CMakeLists.txt